spec/models/piece_spec.rb

```diff
@@ -4,13 +4,140 @@
   describe 'moved flag for piece' do
     it 'should return false if piece has never moved' do
       bishop = FactoryGirl.create(:bishop, :white)
+
       expect(bishop.moved).to eq false
     end
 
     it 'should return true if piece has moved' do
       bishop = FactoryGirl.create(:bishop, :white)
+
       bishop.move!(3, 6)
+
       expect(bishop.moved).to eq true
     end
   end
+
+  describe 'capturable?' do
+    it 'return false if specified coordinates not occupied by any piece' do
+      game = FactoryGirl.create(:game)
```

Comment:
Can you conform the language used for it blocks to the rest of the rspecs so they read consistently.  For example, your it statements should begin with 'should'.